Userland/Libraries/LibAudio/WavWriter.cpp

```diff
@@ -40,10 +40,11 @@ void WavWriter::set_file(StringView path)
     m_finalized = false;
 }
 
-void WavWriter::write_samples(const u8* samples, size_t size)
+ErrorOr<void> WavWriter::write_samples(const u8* samples, size_t size)
```

Comment:
This is perfectly fine.

But now that I see this function signature, I'd like to bait someone into making this accept a `Span<u8>`, just to prevent weird memory issues. (That would be a separate PR. You can immediately mark this "resolved".)

Reply:
That sound interesting, I might look into that, now IODevice is deprecated. I make no promises though